src/kleinanzeigen_bot/__init__.py

```diff
@@ -1195,7 +1208,16 @@ def main(args:list[str]) -> None:
     try:
         bot = KleinanzeigenBot()
         atexit.register(bot.close_browser_session)
-        nodriver.loop().run_until_complete(bot.run(args))
+        while True:
```

Comment:
why is this loop necessary. Isn't that what https://github.com/Second-Hand-Friends/kleinanzeigen-bot/blob/f2e6f0b20bf767765c58d2da4ac61bc93b46ee68/src/kleinanzeigen_bot/__main__.py#L14 is doing?